### PR TITLE
IP and Supply Update

### PR DESCRIPTION
**Pull request by GrassRootsCrypto**
- [x]  Update IP
- [x]  Update RUNE Supply
- [x]  Update Tools and Dashboard Links
- [x] Fix Lending Table
- [x] Fix Broken links, old examples, spelling issues, unnecessary spaces
- [x] Update ecosystem page

### CODE REVIEW

File: frequently-asked-questions/liquidity-providers.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ You can withdraw your symmetrical deposit both asymmetrically (in either asset)
 
 **Will all ERC20 and BEP2 assets be supported and listed on THORChain by default?**
 
-No. Only short tail assets with high MarketCap, good velocity and economic activity would have chances to win liquidity competition to get listed.  Token whitelists being maintained, for security purposes:
+No. Only short tail assets with high MarketCap, good velocity and economic activity would have chances to win liquidity competition to get listed. Token whitelists being maintained, for security purposes:
```
Comment (GrassRootsCrypto) [inline frequently-asked-questions/liquidity-providers.md:56]: This needs to be removed

File: frequently-asked-questions/README.md [hunk 2]
```diff
@@ -20,19 +20,19 @@ All tokens have fully vested since August 2023.
...
+Approx. 330M with a max of 500M. Full details at [https://ops.ninerealms.com/network](https://ops.ninerealms.com/network) or [hhttps://runescan.io/addresses](https://runescan.io/addresses/) for a breakdown of the RUNE supply.
 
 ### **What is the inflation rate of RUNE?**
 
-The protocol [inflation](https://docs.thorchain.org/network/emission-schedule) is perdominally determined by the size of the reserve. The formula is: `Reserve/EmissionCurve/BlocksPerYear/`, thus gradually decreases every block. In early 2024, with an active reserve of 83m RUNE, the dailyinflation is around 28.5k RUNE. See the  [THORChain Tokenomics article](https://medium.com/thorchain/thorchain-tokenomics-what-is-rune-52d339633260) for more information.
+The protocol [inflation](https://docs.thorchain.org/network/emission-schedule) is perdominally determined by the size of the reserve. The formula is: `Reserve/EmissionCurve/BlocksPerYear/`, thus gradually decreases every block. In early 2024, with an active reserve of 83m RUNE, the dailyinflation is around 28.5k RUNE. See the [THORChain Tokenomics article](https://medium.com/thorchain/thorchain-tokenomics-what-is-rune-52d339633260) for more information.
```
Comment (Orion-9R) [inline frequently-asked-questions/README.md:35]: predominantly

Comment (Orion-9R) [reply frequently-asked-questions/README.md:35]: this is fine for now, but going forward let's refrain from having specific values as that just becomes out-dated.

File: frequently-asked-questions/README.md [hunk 3]
```diff
@@ -44,25 +44,25 @@ You can many interfaces to interact with THORChain. See [Exchanges](../ecosystem
 
 #### **What is Outbound Throttling?**
 
-* See [Outbound Transaction Throttling](../how-it-works/security.md#b905-1)
-* You can watch it at [https://thorchain-scheduled-tx.web.app/](https://thorchain-scheduled-tx.web.app/)
-* Since Jan 2024, [Swapper Clout](https://gitlab.com/thorchain/thornode/-/issues/1723) has been enabled to optimise the Outbound Throttling.
+- See [Outbound Transaction Throttling](../how-it-works/security.md#b905-1)
+- You can watch it at [https://thorchain-scheduled-tx.web.app/](https://thorchain-scheduled-tx.web.app/)
```
Comment (Orion-9R) [inline frequently-asked-questions/README.md:48]: broken link?

Comment (Orion-9R) [reply frequently-asked-questions/README.md:48]: could replace with: https://track.ninerealms.com/ Outbound section?

**Pull request merged**